website/pages/en/deploying/deploying-a-subgraph-to-studio.mdx

```diff
@@ -1,17 +1,36 @@
 ---
-title: Deploying a Subgraph to Subgraph Studio
+title: Deploy Using Subgraph Studio
 ---
 
-These are the steps to deploy your subgraph to Subgraph Studio:
+Learn how to deploy your subgraph in Subgraph Studio.
 
-- Install The Graph CLI (with either yarn or npm)
-- Create your Subgraph in Subgraph Studio
-- Authenticate your account from the CLI
-- Deploying a Subgraph to Subgraph Studio
+> Note: When you deploy a subgraph, you push it to Subgraph Studio, where you'll be able to stage it. It's important to remember that deploying is not the same as publishing. When you publish a subgraph, you're publishing it on-chain.
 
-## Installing Graph CLI
+## Subgraph Studio Overview
 
-There is a CLI to deploy subgraphs to [Subgraph Studio](https://thegraph.com/studio/). Here are the commands to install `graph-cli`. This can be done using npm or yarn.
+In [Subgraph Studio](https://thegraph.com/studio/), you can do the following:
+
+- Create and manage your API keys for specific subgraphs
+- Create your subgraph through Studio UI
+- Deploy your subgraph using the CLI
+- Publish your subgraph with the Studio UI
+- Stage your subgraph in the playground environment
+- Integrate your subgraph in staging using the query URL
```

Comment:
```suggestion
- Integrate your subgraph in staging using the development query URL
```

Also, I think this bullet point (and potentially the previous one as well) should come before "Publish your subgraph with the Studio UI", because using the development query URL to test a subgraph in a staging environment is usually done before publishing it to the network. After publishing it, the production query URL should be used.